src/pgrn-list-broken-indexes.c

```diff
@@ -59,18 +59,23 @@ PGrnTableHaveBrokenColumn(grn_obj *table)
 }
 
 static bool
-PGrnIsBrokenSources(Relation index)
+PGrnIsBrokenGrnObj(grn_obj *obj)
```

Comment:
We call `PGrnTableHaveBrokenColumn()`. So we should use `Table` not `GrnObj` here. `GrnObj` is too generic because a column is also a `grn_obj`.

```suggestion
PGrnIsBrokenTable(grn_obj *table)
```